### ISSUE TITLE
Unclear error message in spaced-comment rule

### ISSUE THREAD

**Issue by zandaqo** [eslint/eslint#5030]
The error message of spaced-comment rule in ESLint @ 9a7574f9a41756268de2eaad487d741afc576ad0 does not account for markers:
```javascript
/* eslint spaced-comment: [2, "always"] */

/**This is a comment with a marker but without whitespace  
/*error Expected space or tab after /* in comment.*/
```
The error message leads to believe that there should be a space after `/*` where in reality the marker `*` is used in the example and placing the space after `/**` is equally right and should be preferred.

**Comment by eslintbot** [comment]
@zandaqo Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
Yeah, this would be nice to make things clearer.

**Comment by nzakas** [comment]
Any interest in submitting a PR?

**Comment by zandaqo** [comment]
@nzakas I did: #5031

I just bundled it with fix for #4969 since a) this one is a small issue; b) this way both issues are fix with addition of just one regexp to pick up markers.
